backend/src/test/java/reviewme/member/domain/ReviewerGroupTest.java

```diff
@@ -19,32 +24,83 @@ class ReviewerGroupTest {
+        Member sancho = 회원_산초.create();
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of();
+
         // when, then
-        assertThatThrownBy(() -> new ReviewerGroup(sancho, List.of(), groupName, "설명", createdAt))
+        assertThatThrownBy(() -> new ReviewerGroup(sancho, githubIds, groupName, "설명", createdAt))
                 .isInstanceOf(InvalidGroupNameLengthException.class);
     }
 
     @Test
-    void 리뷰_그룹_설명_길이_제한을_벗어나는_경우_예외를_발생한다() {
+    void 리뷰_그룹_설명_길이_제한을_벗어날_수_없다() {
         // given
         String description = "a".repeat(51);
-        Member sancho = new Member("산초", 1);
-        LocalDateTime createdAt = LocalDateTime.of(2024, 7, 17, 12, 0);
+        Member sancho = 회원_산초.create();
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of();
+
         // when, then
-        assertThatThrownBy(() -> new ReviewerGroup(sancho, List.of(), "그룹 이름", description, createdAt))
+        assertThatThrownBy(() -> new ReviewerGroup(sancho, githubIds, "그룹 이름", description, createdAt))
                 .isInstanceOf(DescriptionLengthExceededException.class);
     }
+
+    @Test
+    void 리뷰어_목록에_리뷰이가_들어갈_수_없다() {
+        // given
+        Member member = 회원_산초.create();
+        String groupName = "Group";
+        String description = "Description";
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> reviewerGithubIds = List.of(member.getGithubId());
+
+        // when, then
+        assertThatThrownBy(() -> new ReviewerGroup(member, reviewerGithubIds, groupName, description, createdAt))
+                .isInstanceOf(SelfReviewException.class);
+    }
+
+    @Test
+    void 리뷰어_목록이_비어있을_수_없다() {
+        Member member = 회원_산초.create();
+        String groupName = "Group";
+        String description = "Description";
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of();
+
+        // when, then
+        assertThatThrownBy(() -> new ReviewerGroup(member, githubIds, groupName, description, createdAt))
+                .isInstanceOf(EmptyReviewerException.class);
+    }
+
+    @Test
+    void 리뷰어를_중복으로_추가할_수_없다() {
+        // given
+        Member reviewee = 회원_산초.create();
+        Member reviewer = 회원_커비.create();
+
+        String groupName = "Group";
+        String description = "Description";
+        LocalDateTime createdAt = LocalDateTime.now();
+        List<GithubId> githubIds = List.of(reviewer.getGithubId());
+        ReviewerGroup reviewerGroup = new ReviewerGroup(reviewee, githubIds, groupName, description, createdAt);
+        GithubIdReviewerGroup githubIdReviewerGroup = new GithubIdReviewerGroup(reviewee.getGithubId(), reviewerGroup);
+        // when, then
+        assertThatThrownBy(() -> reviewerGroup.addReviewerGithubId(githubIdReviewerGroup))
+                .isInstanceOf(DuplicateReviewerException.class);
```

Comment:
리뷰어를 중복 등록할 수 없는 거면 reviewer를 갖고 githubIdReviewerGroup를 만드는 게 맞지 않을까요?
```suggestion
        GithubIdReviewerGroup githubIdReviewerGroup = new GithubIdReviewerGroup(reviewer.getGithubId(), reviewerGroup);

        // when, then
        assertThatThrownBy(() -> reviewerGroup.addReviewerGithubId(githubIdReviewerGroup))
                .isInstanceOf(DuplicateReviewerException.class);
```

Reply:
오 이건 한 사람 추가할 때의 검증이라, 만들 때의 검증도 추가할게요!